src/Auth/Startup.cs

```diff
@@ -76,19 +97,35 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseExceptionHandler("/Home/Error");
             }
 
-            app.UseIdentity();
+            //app.UseIdentity();
```

Comment:
We don't need this line as it is for adding cookie base authentication for request pipeline